src/api/app/views/layouts/webui/responsive_ux/_footer.html.haml

```diff
@@ -0,0 +1,43 @@
+.d-flex
+  - if User.session
+    .flex-fill.d-none.d-md-block
+      %strong.text-uppercase User
+      %ul
+        %li= link_to('Your Profile', user_path(User.session!))
+        - if User.session!.home_project
+          %li= link_to('Home Project', project_show_path(User.session!.home_project))
+        - else
+          %li= link_to('Create Home Project', new_project_path(name: User.session!.home_project_name))
+        %li= link_to('Logout', session_path, method: :delete)
+  - if ::Configuration.anonymous
```

Comment:
Hm, why?

Reply:
Removed. It was included in the old code. But yes, every link is checked in a before_action, so no need to hide this links.